buh/streams.yaml

```diff
@@ -0,0 +1,44 @@
+proxima.eth-main.blocks.1_0:
```

Comment:
the idea is to use one single file `streams.yaml` in the repo root for all streams without giving details where it's located